### ISSUE TITLE
Whitespace in Markdown (enters)

### ISSUE THREAD

**Issue by monotkate** [synergistic-silobusters/flintandsteel#336]
If you submit a comment with enters between lines of text, it eliminates it after it's submitted.

![image](https://cloud.githubusercontent.com/assets/11742296/13477192/2a71792a-e099-11e5-983d-dc792a8c5d83.png)

![image](https://cloud.githubusercontent.com/assets/11742296/13477204/3ca7a844-e099-11e5-829b-d13ebe3eff39.png)

**Comment by Mctalian** [comment]
That's because comments aren't Markdown enabled. It works fine in
description and updates.

**Comment by Mctalian** [comment]
So, this can become an enhancement request and/or tech spike to enable only
a small subset of Markdown for comments. It might, however, screw up all
other Markdown if it's a global configuration.

**Comment by monotkate** [comment]
Yeah I know it's not in markdown but that's weird behavior. Other solution would be to return to a submit on enter for comments

**Comment by Mctalian** [comment]
I agree it's weird. It's a limitation of HTML, unfortunately. I posted in some issue all the ways I know of to get around this sort of thing, I'll see if I can find it. Enabling Markdown was the solution we chose, but I would vote against Markdown on comments because people could use giant headers and make comments super difficult to scroll through.

**Comment by Mctalian** [comment]
#264 my comment lists the ways I'm aware of working around it. In addition we can try to create a custom markdown directive with limited features. Perhaps just hyperlinks, lists, and newlines to name a few.

**Comment by monotkate** [comment]
How about submit on enter?

**Comment by Mctalian** [comment]
Yeah, that's another option. I know when Yammer submits on enter I get pissed though.
